src/block-card.css

```diff
@@ -0,0 +1,91 @@
+.card {
+  display: flex;
+  flex-direction: column;
+  background-color: #fff;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+  width: max-content;
+}
+
+.card__image-container {
+  margin-bottom: 40px;
+  display: flex;
+  padding: 0 3px;
+}
+
+.card__image {
+  height: 134px;
+  width: 160px;
+}
+
+.card__name {
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+  color: #060b35;
```

Comment:
Move repeatable colors to CSS variables and move them to :root selector.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)